### DISCUSSION TITLE
How to set classpath when run Detekt from CLI to use Type Resolution

### DISCUSSION THREAD

**Discussion by oandrz** [detekt/detekt#4945] [Q&A] [answered]
Hi I want to ask small question, how to set classpath when run Detekt from CLI to use Type Resolution ? What are the value that we set to there ?

**Comment by cortinico** [comment, marked as answer]
You can use the `-cp` parameter on the CLI: https://detekt.dev/docs/gettingstarted/cli/#use-the-cli

**Reply by oandrz** [reply]
@cortinico Ah sorry I think my questoin is unclear, I mean what is the value of the classpath that we set ? do we have some kind of example ?

**Reply by cortinico** [reply]
Not really, as we recommend to use the Gradle Plugin for feeding the classpaths. Theoretically you should be good by passing a semi-color separate list with all of your dependencies: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/classpath.html
